src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -159,52 +81,74 @@ const ChatItemMyTextV2 = ({
+    if (direction === 'right') return;
+    if (swipeableRef.current) swipeableRef.current?.close();
+    setReplyPreview({
+      username,
+      time,
+      message,
+      messageId: data?.id,
+      chatType,
+      messageType: 'regular'
+    });
   };
 
   return (
-    <View style={styles.chatContainer}>
-      <View style={styles.wrapper}>
-        <ContextMenu
-          previewBackgroundColor="transparent"
-          style={{flex: 1}}
-          actions={[
-            {title: 'Reply', systemIcon: 'arrow.turn.up.left'},
-            {title: 'Copy Message', systemIcon: 'square.on.square'},
-            {title: 'Delete Message', systemIcon: 'trash', destructive: true}
-          ]}
-          onPress={(e) => onContextMenuPressed(e, message)}>
-          <View style={{borderRadius: 8}}>
-            <View style={handleTextContainerStyle()}>
-              {!isContinuous && (
-                <View style={styles.chatTitleContainer}>
-                  <Text style={styles.userText}>{username}</Text>
-                  <View style={styles.dot} />
-                  <Text style={styles.timeText}>{time}</Text>
-                </View>
-              )}
-
-              <Text ref={messageRef} style={styles.text} onTextLayout={onTextLayout}>
-                {`${message}`}
-              </Text>
-
-              {renderIcon()}
+    <Swipeable
+      testID="swipeable"
+      ref={swipeableRef}
+      friction={3}
+      overshootFriction={2}
+      onSwipeableOpen={onSwipeToReply}
+      renderLeftActions={replyIcon}>
+      <View style={containerStyle(true, isReplyPrompt)}>
+        <Animated.View style={[styles.wrapper, animatedBubbleStyle]}>
+          <ContextMenu
+            previewBackgroundColor="transparent"
+            style={{flex: 1}}
+            actions={contextMenuActions}
+            onPress={(e) => onContextMenuPressed(e, data, chatType)}>
+            <View style={styles.radius8}>
+              <View style={textContainerStyle(true, chatType, isNewLine)}>
+                {isShowUserInfo && (
+                  <View style={styles.chatTitleContainer}>
+                    <Text style={[styles.userText, textStyle(true)]}>{username}</Text>
+                    <View style={dotStyle(true)} />
+                    <Text style={[styles.timeText, textStyle(true)]}>{time}</Text>
+                  </View>
+                )}
+
+                <ChatReplyView
+                  type={chatType}
+                  messageType={messageType}
+                  replyData={data?.reply_data}
+                />
+
+                <Text style={messageStyle(true, messageType)} onTextLayout={onTextLayout}>
+                  {`${message}`}
+                </Text>
+
+                {renderIcon()}
+              </View>
             </View>
-          </View>
-        </ContextMenu>
+          </ContextMenu>
 
-        {renderAvatar()}
+          {renderAvatar()}
+        </Animated.View>
       </View>
-    </View>
+    </Swipeable>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the new hunk, you've introduced swipeable functionality with the `Swipeable` component. However, you're only handling the `onSwipeableOpen` event. Consider also handling the `onSwipeableClose` event if necessary, to reset any state or UI changes made during the swipe.

Also, you're using `ContextMenu` to provide additional options for each chat item. Ensure that the `onPress` function `onContextMenuPressed` properly handles all the possible actions and their effects on the application state.

Lastly, you've added a `ChatReplyView` component which wasn't present in the old hunk. Make sure that this component is properly tested and doesn't introduce any regressions in the chat functionality.


<!-- This is an auto-generated comment by OSS CodeRabbit -->